migrations/20200917163300-create-events.js

```diff
@@ -0,0 +1,19 @@
+exports.up = (db, callback) => db.createTable('event', {
+  id: { type: 'int', primaryKey: true, autoIncrement: true },
+  type: 'string',
+  category: 'string',
+  name: 'string',
+  model: 'string',
+  modelId: 'int',
+  createdAt: 'timestamp',
+  updatedAt: 'timestamp',
+  hiddenAt: 'timestamp',
+  body: 'jsonb',
+})
+  .then(() => db.runSql('CREATE INDEX "idxEventBody" ON site USING gin ("body");'))
```

Comment:
Is this index correct? What is it for?

Reply:
this indexes the json so it is more efficiently searched?

Reply:
this is indexing on a key named `site` in the json? Will we be searching by this alot?

Reply:
the index on the json attribute ... i've found it helpful in the past for querying jsonb fields ... however,  if your concern is that you don't think we'll use the index often to search, i'm okay with opting out of it now and we can add it later if we need it?